feature_engine/datasets/data.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+import numpy as np
+
+
+def load_titanic(return_X_y_frame=False, predictors_only=False, raw=False):
+    df = pd.read_csv("https://www.openml.org/data/get_csv/16826755/phpMYEkMl")
+
+    if predictors_only:
+        df.drop(
+            columns=["name", "ticket", "home.dest", "boat", "body", "cabin"],
+            inplace=True,
+        )
+
+    if not raw:
```

Comment:
maybe we could call this param `handle_missing`, and if True, impute them (by mean in numerical and mode in  categorical), and if False, return them as np.nan. I don't think we need the "?" at all, or do we?

Reply:
I like it...

Reply:
@solegalli 
In order to implement imputation here, I used `CategoricalImputer` with `encoding_method="frequent"` to impute mode. But when doing this I get the error that `body` has multiple frequent values. So my question is, this is an expected behavior for `CategoricalImputer`? How are the ties addressed in this Imputation?

Reply:
@solegalli I did a different thing. Since imputation can be applied to `predictors_only` True or False, I thought it could be a better idea to impute categories with a missing indicator. The reason of this, is since not predictor variables have a high number of missing values and imputing mode it could be not quite correct. What do you think?

Reply:
I think you did a great job here. I wouldn't change it. Thank you for picking this up.